src/content_sao.cpp

```diff
@@ -385,6 +385,13 @@ std::string LuaEntitySAO::getClientInitializationData(u16 protocol_version)
 			os<<serializeLongString(gob_cmd_update_bone_position((*ii).first, (*ii).second.X, (*ii).second.Y)); // m_bone_position.size
 		}
 		os<<serializeLongString(gob_cmd_update_attachment(m_attachment_parent_id, m_attachment_bone, m_attachment_position, m_attachment_rotation)); // 4
+		for (std::set<int>::const_iterator ii = m_attachment_child_ids.begin(); 
```

Comment:
can you use UNORDERED_SET instead of std::set and change the container type of m_attachment_child_ids to this too ?